hypothesis-python/docs/_static/no-scroll.css

```diff
@@ -0,0 +1,5 @@
+/* disable autoscroll-to-target behavior
+   https://github.com/pradyunsg/furo/discussions/384#discussioncomment-2249243 */
+html {
+    scroll-behavior: auto;
+}
```

Comment:
I don't know if I'm in a minority that's bothered by this or what, but the scroll animation makes the docs feel sluggish imo.

Reply:
I am _very strongly_ on board with "never have js do the browser's job", and scrolling is definitely a browser job.